src/main/java/ru/nms/exceptions/BallTreeFileFormatMismatchException.java

```diff
@@ -0,0 +1,11 @@
+package ru.nms.exceptions;
+
+import static ru.nms.utils.Constants.BALL_TREE_HEADER;
+
+public class BallTreeFileFormatMismatchException extends RuntimeException{
```

Comment:
```suggestion
public class BallTreeFileFormatMismatchException extends RuntimeException {
```

Более глобальный совет, сразу подключать какой-то линтер в проекте и CI (офк, это за рамками лабы, но всяко полезно упомянуть)